### PR TITLE
added share screen user docs + gifs

### PR DESCRIPTION
**Pull request by liam-philiben**
First pass at the user docs for share screen w/ relevant GIFs! lmk if you find any grammatical errors or typos, any issues or comments with the gifs/images, or general feedback on the overall flow of info 😅

### CODE REVIEW

File: docs/WebEditorBasics/SharingExperiences.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+![Collaborator email found](/img/product/editor/sharing/sharing_enklu_users.png)
+
+### Inviting Non-Enklu Users
+
+If a collaborator's email address is not currently associated with an Enklu account, you will be able to send an email invite. This email invite will allow the recipient to create an Enklu account and share your selected experience with them once their account has been created.
+
+![Collaborator email not found](/img/product/editor/sharing/sharing_non_enklu_users.gif)
+
+### Removing a Collaborator
```
Comment (ShannonKalei) [inline docs/WebEditorBasics/SharingExperiences.md:27]: Not a big deal, but this feels more like an h2 than an h3

**Review by ShannonKalei — APPROVED** [review]
Digging all the evidence that it was indeed Liam who was changing people's permissions during the playtest 😛 

`placeholder.png` I didn't see used and sure sounds like something that can be removed.  

Also making a mental note for myself that there's still so many old editor images we need to replace with our new dark mode 😅

File: docs/WebEditorBasics/SharingExperiences.md (hunk 1 above)
Comment (liam-philiben) [reply docs/WebEditorBasics/SharingExperiences.md:27]: Re: placeholder.png, that was just the Enklu logo that I used as, well, a placeholder before I had actual gifs and pngs haha. I can remove it tho

File: sidebars.json [hunk 2]
```diff
@@ -5,6 +5,7 @@
       "WebEditorBasics/WebEditorInterface",
       "WebEditorBasics/MinimumRequirements",
       "WebEditorBasics/ManagingYourExperiences",
+	  "WebEditorBasics/SharingExperiences",
```
Comment (gdamron) [inline sidebars.json:8]: Small thing: please make it so your text editor converts tabs to spaces and uses 2 spaces for indentation.

**Review by gdamron — APPROVED** [review]
just a couple of minor suggestions

File: sidebars.json (hunk 2 above)
Comment (liam-philiben) [reply sidebars.json:8]: done!

Comment (liam-philiben) [reply sidebars.json]: hopefully it looks properly formatted now

File: docs/WebEditorBasics/SharingExperiences.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+If you are the owner of an experience, or are an Editor/Admin for one, you can edit a collaborator's permission level via a drop down menu in the main modal dialog box.
+
+![Editing Permissions](/img/product/editor/sharing/editing_permissions.png)
+
+You will then be able to select the following permission levels for that collaborator:
+> - **Viewer:** Has access, but cannot edit or share the experience.
+> - **Editor:** Able to edit content, and can add or remove Viewers or Editors.
+> - **Admin:** Highest permission level, can do everything an Editor can, as well as delete an experience entirely.
+
+You can only change a person's permission level up to your own level of permissions. This means that while an Editor cannot elevate a Viewer to Admin, but an Admin can promote a Viewer or Editor to Admin status.
```
Comment (jsundra) [inline docs/WebEditorBasics/SharingExperiences.md:55]: Unnecessary "but" in the second sentence.

**Review by jsundra — APPROVED** [review]
Wow, really showing off that you dropped me down to Viewer, huh? Looks good overall, a few suggestions here and there.

**Pull request merged**